parquet-column/src/main/java/org/apache/parquet/io/vector/ByteColumnVector.java

```diff
@@ -0,0 +1,38 @@
+/* 
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.io.vector;
+
+import org.apache.parquet.column.ColumnReader;
+import org.apache.parquet.io.ColumnVector;
+
+public class ByteColumnVector extends ColumnVector
+{
+  public final byte[][] values;
```

Comment:
Why not use `Binary[]`? Using `getBytesUnsafe` will copy the underlying bytes in many cases; for example values from a dictionary or other underlying binary stream are wrapped in `Binary` using the array slice method, which always copies in this situation.